thirdeye/thirdeye-frontend/app/pods/home/controller.js

```diff
@@ -0,0 +1,4 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+});
```

Comment:
Empty controller, just making sure you didn't mean to add something here lol

Reply:
I believe the intention was to create it now since we'll need it eventually ?

Reply:
Yes, @justYves is correct.